gatsby-browser.js

```diff
@@ -1,20 +1,20 @@
 /* eslint-disable import/prefer-default-export */
-
+/* eslint-disable import/no-extraneous-dependencies */
 
 import { createElement } from 'react';
 import { Provider } from 'react-redux';
-
+import * as Sentry from '@sentry/browser';
 
 import { createUser } from './src/redux/actions';
 import store from './src/redux/store';
 
-
 const wrapRootElement = ({ element }) => (
   createElement(Provider, { store }, element)
 );
 
-
 const onClientEntry = () => {
+  Sentry.init({ dsn: 'https://4beb31b94ed742d9aeeb981a354fc690@sentry.io/1416880' });
```

Comment:
What Sentry project is this? It seems to be a different one than https://sentry.io/settings/openup-sa/projects/khetha-react/ ?

Reply:
Okay, we sorted out the duplicate Sentry org / project creation.

Reply:
Project: `khetha-react`, https://sentry.io/organizations/openupsa/issues/?project=1416880